test/distributed/test_data_parallel.py

```diff
@@ -855,5 +769,102 @@ def forward(self, inp):
             model(input)
 
 
+class TestDataParallelDeviceType(TestCase):
+
+    @onlyCUDA
+    @skipMeta
+    @dtypes(*ALL_TENSORTYPES)
```

Comment:
This should also stop using this "ALL_TENSORTYPES" list and just write in the dtypes in question or use one of the ATen macro dispatch list functions.